Internship.API/Controllers/InternsController.cs

```diff
@@ -159,5 +159,15 @@ public ActionResult<InternDTO> Update(string id, InternDTO internIn)
                 return BadRequest(new ApiError(400, "Request failed", e.Message));
             }
         }
+        /// <summary>
+        /// Get all interns registered
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Can you add documentation about what is this method returning?

Reply:
I did it.